src/riot-rs-embassy/src/sendcell.rs

```diff
@@ -40,12 +40,13 @@ impl<T> SendCell<T> {
     }
 
     /// Gets the contents of this [`SendCell`].
-    pub fn get(&self, spawner: Spawner) -> Option<&T> {
-        if spawner.executor_id() == self.executor_id {
-            Some(&self.inner)
-        } else {
-            None
-        }
+    pub fn get(&self, _spawner: Spawner) -> Option<&T> {
+        Some(&self.inner)
+        // if spawner.executor_id() == self.executor_id {
+        //     Some(&self.inner)
+        // } else {
+        //     None
+        // }
```

Comment:
The `threading-net-example` initialized the stack in one thread (the `esp_wifi_thread` thread that initialized embassy) and accesses it in the seconds thread in `tcp_echo`.

Reply:
`SendCell` is to prevent non-Send objects to be sent across executors, which is usually prevented by the type system. Removing this check breaks the `unsafe` assumptions for the `unsafe impl Send for SendCell`.

Reply:
Yes; I just removed it as a quick fix because I didn't know how else to solve this. Any ideas?

Reply:
Outdated with https://github.com/future-proof-iot/RIOT-rs/pull/300#issuecomment-2129033999.